sw/device/silicon_creator/lib/boot_svc/boot_svc_header.h

```diff
@@ -0,0 +1,67 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_BOOT_SVC_BOOT_SVC_HEADER_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_BOOT_SVC_BOOT_SVC_HEADER_H_
+
+#include <stdint.h>
+
+#include "sw/device/lib/base/macros.h"
+#include "sw/device/silicon_creator/lib/base/chip.h"
+#include "sw/device/silicon_creator/lib/drivers/hmac.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+/**
+ * Boot services message header.
+ *
+ * All boot services messages start with a common header followed by a message
+ * specific payload.
+ */
+typedef struct boot_svc_header {
+  /**
+   * SHA256 digest of the message.
+   *
+   * Digest region starts at `identifier` and extends until the end of the
+   * message.
+   */
+  hmac_digest_t digest;
+  /**
+   * Identifier.
+   *
+   * 0x43565342 (ASCII "BSVC").
```

Comment:
Nit: maybe just refer to `kBootSvcIdentifier` here rather than hardcoding its value into the comment.

Reply:
You're right. Done.